packages/lib-classifier/src/store/SubjectStore.js

```diff
@@ -152,17 +152,25 @@ const SubjectStore = types
       const root = getRoot(self)
       const client = root.client.panoptes
       const workflow = tryReference(() => root.workflows.active)
+      let apiUrl = '/subjects/queued'
+      
       if (workflow) {
         self.loadingState = asyncStates.loading
         const params = { workflow_id: workflow.id }
         if (workflow.grouped) {
           params.subject_set_id = workflow.subjectSetId
         }
+        
+        if (workflow.configuration.subject_viewer === 'subjectGroup') {
+          apiUrl = '/subjects/grouped'
```

Comment:
Should we have a short ADR so that we don't forget that the API endpoint is now configurable?

Reply:
This definitely needs a follow up ADR, and an expansion of the README now that the endpoint has more or less been settled.